content/docs/getting-started/user-management/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: User Managment in Backstage
+lastUpdated: '2021-09-21T14:00:00.0Z'
+description: How to manage users in Backstage
+---
+
+## Introduction
+
```

Comment:
Might be worth adding the caveat that this only works if you are using auth0

Reply:
I put the caveat at the bottom of the page, so as not to interupt the flow, but I am also undecided. :/

Reply:
Setup documentation is mostly used by people in free trials. All of the free trial people will be using Auth0. So most of the readers of this document will be using Auth0. For that reason, I like Brian's layout, with the comment about SSO at the bottom of the page.

Reply:
Alright cool. sure lets go with it the way it is.